lib/v2/baidu/router.js

```diff
@@ -1,7 +1,7 @@
 module.exports = (router) => {
     router.get('/gushitong/index', require('./gushitong'));
-    router.get('/tieba/forum/:kw', require('./tieba/forum'));
-    router.get('/tieba/forum/good/:kw/:cid?', require('./tieba/forum'));
+    router.get('/tieba/forum/:kw/:sortBy?', require('./tieba/forum'));
+    router.get('/tieba/forum-good/:kw/:cid?/:sortBy?', require('./tieba/forum'));
```

Comment:
The route matches from top to bottom, you don't need to use `forum-good`. All you need is to put /forum/good/* before /forum/:kw